src/fortran/client.F90

```diff
@@ -63,7 +63,7 @@ module smartredis_client
                               unpack_tensor_float, unpack_tensor_double
 
   !> Initializes a new instance of the SmartRedis client
-  procedure :: initialize
+  procedure :: initialize_client
```

Comment:
Sorry I think I was a little unclear. I meant that here we would map the procedure `initialize` to `initialize_client.` So that we can still do something like
```
call client_instance%initialize()
```
as opposed to
```
call client_instance%initialize_client()
```
which seems a bit verbose.

So change this line to:
```
  procedure :: initialize => initialize_client
```

Reply:
Thanks for the suggestion. It's implemented and ready for review. I agree that it makes the code better to read

Reply:
Great thanks @billschereriii!